Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -206,17 +216,36 @@ void WriteParameters(NetworkWriter writer, bool autoSend)
                 AnimatorControllerParameter par = parameters[i];
                 if (par.type == AnimatorControllerParameterType.Int)
                 {
-                    writer.WritePackedUInt32((uint)m_Animator.GetInteger(par.nameHash));
+                    int val = m_Animator.GetInteger(par.nameHash);
```

Comment:
please call it 'value'. use proper names for variables so there is no magic :)

Reply:
Did both, thanks.